app/components/searchable_collection_component/searchable_collection_component.html+checkbox.slim

```diff
@@ -1,15 +1,16 @@
-.searchable-collection-component class="#{border_class} #{scrollable_class}"
-  - if searchable
-    .searchable-collection-component__search
-      input.govuk-input.searchable-collection-component__search-input.icon.icon--left.icon--search placeholder="Search"
-      button.searchable-collection-component__search-button type="submit" aria-label="search options"
+= tag.div(class: classes, **html_attributes) do
+  div class="#{border_class} #{scrollable_class}"
```

Comment:
does it work if you put border_class and scrollable_class into `default_classes`? or is that a misuse of 'default'?

Reply:
connor suggested once that if they are dependent on conditions it is a misuse of default, which i am inclined to agree with. but its much of a muchness tbh

Reply:
I think old connor is right.